sources/api/sundog/src/main.rs

```diff
@@ -350,7 +350,14 @@ where
         // Match on the generator's exit code. This code lays the foundation
         // for handling alternative exit codes from generators.
         match result.status.code() {
-            Some(0) => {}
+            Some(0) => {
+                if !result.stderr.is_empty() {
+                    let cmd_stderr = String::from_utf8_lossy(&result.stderr);
+                    for line in cmd_stderr.lines() {
+                        info!("{}: {}", command, line);
```

Comment:
Nit: perhaps the log could be a little bit more verbose so we know exactly what we're looking at in the logs. It could get a little confusing in the context of the whole journal.
```
info!("Setting generator command '{}' stderr: {}", command, line);
```